<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SSPL-LPTP-374" failures="1" tests="3" name="Test" time="11.024" errors="0" timestamp="2025-06-27T10:14:53 IST">
  <testcase classname="testlib.servervalidation.ServerResponsesValidation" name="validateServer" time="7.729">
    <failure type="java.lang.Exception" message="Server is down for https://automation.academiaerp.com">
      <![CDATA[java.lang.Exception: Server is down for https://automation.academiaerp.com
at testlib.servervalidation.ServerResponsesValidation.validateServer(ServerResponsesValidation.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- validateServer -->
  <testcase classname="testlib.servervalidation.ServerResponsesValidation" name="validateServer" time="1.208"/>
  <testcase classname="testlib.servervalidation.ServerResponsesValidation" name="validateServer" time="1.22"/>
</testsuite> <!-- Test -->
